src/pages/Office/CustomerOnboarding/CreateCustomerForm.test.jsx

```diff
@@ -742,4 +742,20 @@ describe('CreateCustomerForm', () => {
       });
     });
   }, 50000);
+
+  it('show bluebark question when BLUEBARK FF is on', async () => {
+    createCustomerWithOktaOption.mockImplementation(() => Promise.resolve(fakeResponse));
+    isBooleanFlagEnabled.mockImplementation(() => Promise.resolve(true));
+    searchLocationByZipCityState.mockImplementation(mockSearchPickupLocation);
+
+    render(
+      <MockProviders initialState={serviceCounselorState}>
+        <CreateCustomerForm {...testProps} />
+      </MockProviders>,
+    );
+
+    await waitFor(() => {
+      expect(screen.getByText('Is this a Bluebark move?')).toBeInTheDocument();
+    });
+  }, 50000);
 });
```

Comment:
Can you add a test to make sure this question does not appear when the FF is off? Should be easy, just change the mocked boolean flag to false and the expect at the end. Im thinking something like this!


```suggestion
  it('Doesnt show bluebark question when BLUEBARK FF is off', async () => {
    createCustomerWithOktaOption.mockImplementation(() => Promise.resolve(fakeResponse));
    isBooleanFlagEnabled.mockImplementation(() => Promise.resolve(true));
    searchLocationByZipCityState.mockImplementation(mockSearchPickupLocation);

    render(
      <MockProviders initialState={serviceCounselorState}>
        <CreateCustomerForm {...testProps} />
      </MockProviders>,
    );

    await waitFor(() => {
      expect(screen.getByText('Is this a Bluebark move?')).toBeInTheDocument();
    });
  }, 50000);
```

Reply:
done